content/influxdb/v2.0/reference/cli/influx/v1/auth/create.md

```diff
@@ -0,0 +1,37 @@
+---
+title: influx v1 auth create
+description: >
+  The `influx v1 auth create` command creates an authorization in the InfluxDB 1.x compatibility API.
+menu:
+  influxdb_2_0_ref:
+    name: influx v1 auth create
+    parent: influx v1 auth
+weight: 101
+influxdb/v2.0/tags: [authorization]
+---
+
+The `influx v1 auth create` command creates a legacy authorization with the [InfluxDB 1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
```

Comment:
Can you create an "all access" token with this command? If yes, which flags need to be set for that?